chromium_src/chrome/browser/ui/views/permission_bubble/chooser_bubble_ui.cc

```diff
@@ -0,0 +1,79 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/brave_wallet/brave_wallet_tab_helper.h"
+#include "brave/common/webui_url_constants.h"
+#include "chrome/browser/profiles/profile.h"
+#include "chrome/browser/ui/browser.h"
+#include "chrome/browser/ui/browser_finder.h"
+#include "chrome/browser/ui/tabs/tab_strip_model.h"
+#include "content/public/browser/web_contents.h"
+#include "ui/views/bubble/bubble_dialog_delegate_view.h"
+#include "ui/views/focus/focus_manager.h"
+#include "ui/views/widget/widget.h"
+#include "url/gurl.h"
+
+namespace {
+
+bool IsBravePanel(content::WebContents* content) {
+  return content->GetURL().EqualsIgnoringRef(GURL(kBraveUIWalletPanelURL));
```

Comment:
https://source.chromium.org/chromium/chromium/src/+/main:content/public/browser/web_contents.h;l=329
<pre>  
// Gets the URL that is currently being displayed, if there is one.
// This method is deprecated. DO NOT USE! Pick either |GetVisibleURL| or
// |GetLastCommittedURL| as appropriate.
</pre>

Reply:
done